src/test/groovy/com/github/rodionovsasha/commentservice/services/CommentServiceTest.groovy

```diff
@@ -66,4 +71,49 @@ class CommentServiceTest extends BaseTest {
         then:
         thrown(TopicNotFoundException)
     }
+
+    def "update updates a comment"() {
+        given:
```

Comment:
Perhaps we can show that the comment changed more explicitly and with less hardcode:
```java
given:
def oldContent = repository.getOne(COMMENT_ID).content

when:
commentService.update(COMMENT_ID, HOMER_ID, COMMENT_CONTENT)

then:
def content = repository.getOne(COMMENT_ID).content
content == COMMENT_CONTENT

and:
content != oldContent
```